src/lib/liveTasks.test.ts

```
@@ -0,0 +1,207 @@
+import { describe, it, expect } from "vitest";
+import {
+  isTaskClaimable, slotsRemaining, isUserAssigned,
```

ESLint

[error] Replace `·slotsRemaining,` with `⏎··slotsRemaining,⏎·`

(prettier/prettier)